src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -801,6 +867,15 @@ string getRuleActionCore(bool enable)
             }
         }
 
+        private static void Validate(string fileWithPath, string fileContents, ref bool validationResult)
```

Comment:
Instead of `ref bool validationResult`, we can use `List<string> fileNamesWithValidationFailures` or such and this method just adds to the list. Display is done by the main method by iterating through these file names.